src/_http/index.js

```diff
@@ -131,6 +142,29 @@ HttpClient.prototype.execute = function(options) {
   })
 }
 
+function isValidTraceparentHeader(traceparentHeader) {
+  // Shamelessly copied from shorturl.at/osvwz
+  return /^[\da-f]{2}-[\da-f]{32}-[\da-f]{16}-[\da-f]{2}$/.test(
+    traceparentHeader
+  )
+}
+
+function parseTags(tags) {
+  if (tags === undefined || tags == null || tags == '') return null
+  if (typeof tags != 'object') {
+    throw new Error('Tags must be provided as an object!')
+  }
+  var result = ''
+  const entries = Object.entries(tags)
+  const entriesLength = entries.length
+  var index = 1
+  entries.forEach(([key, value]) => {
+    result += key + '=' + value
+    if (index++ < entriesLength) result += ','
+  })
+  return result
```

Comment:
```javascript
return Object.entries(tags).map(e => e.join('=')).join()
```

Reply:
> ```js
> return Object.entries(tags).map(e => e.join('=')).join()
> ```

Missing ',' from the final `join`. Seems like it should be:

```
return Object.entries(tags).map(e => e.join('=')).join(',')
```

Reply:
comma is the default, but yes better to be verbose

Reply:
```javascript
return Object.entries(tags).map(e => e.join('=')).join(',')
```

Much cleaner, thanks.

>maybe we want to validate the key:value pairs
    silently throw out bad ones

Exception is preferred to silently discarding. The only reason we silently discard invalid traceparents is because the W3C spec defines it that way, but in general, we should try to consider customer requests immutable. That said, let me sync with product on a reasonable length of tags and update that in a separate PR.